lib/propshaft/railtie.rb

```diff
@@ -15,6 +15,11 @@ class Railtie < ::Rails::Railtie
       [ "text/css", Propshaft::Compilers::SourceMappingUrls ],
       [ "text/javascript", Propshaft::Compilers::SourceMappingUrls ]
     ]
+    if Rails.env.development?
+      config.assets.cache_control_header = "no-store"
+    else
+      config.assets.cache_control_header = "public, max-age=31536000, immutable"
```

Comment:
Does it make sense to have this value out-of-sync with `config.assets.sweep_cache`?
Wouldn't be easier to keep it in sync by default like

```ruby
config.assets.cache_control_header = config.assets.sweep_cache ? "no-store" : "public, max-age=31536000, immutable"
```

:question:

Reply:
yep  `sweep_cache` makes more sense 👍, I will change it later this week 😇

Reply:
Clear, this could be fixed by setting original value of sweep_cache to `nil` and only when it is not assigned to false explicitly, change `cache_control_header` and also self assign to `false`. Anything in here works for me. Even two independent config options as originally proposed. With sane default values per environment it would be rarely tweaked by users anyway.

Reply:
Sorry I'm getting back after 2 weeks (I had some personal issues to solve) 

Thank you for the recommendation how to fix this with `nil` `sweep_cache`. I understand 

>  Even two independent config options as originally proposed

Unless you really  want the sweep_cache nil  solution (as you proposed) I would vote for the original "two independent config options" solution.  I get it no-one will really change the values.  But personally purely to have simple straight forward  config. 

please let me know if I should do the `nil` change proposed or not

Reply:
As mentioned no huge preference from my side. Let's grab some maintainers attention somehow, they can decide on their own. :pray:

@brenogazzola @dhh friendly ping